pyproject.toml

```diff
@@ -1,3 +1,35 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "xcdat"
+version = "0.7.3"
```

Comment:
You might want to make this dynamic:
https://github.com/E3SM-Project/mache/blob/main/pyproject.toml#L74-L75

Reply:
https://github.com/E3SM-Project/mache/blob/main/pyproject.toml#L3

Reply:
I added your suggestion, thanks.